Packs/F5Silverline/Integrations/F5Silverline/F5Silverline.py

```diff
@@ -186,9 +188,11 @@ def delete_ip_objects_command(client: Client, args: Dict[str, Any]):
         human_readable = ""
         for object_id in object_id_list:
             url_suffix = f'{list_type}/ip_objects/{object_id}'
-            client.request_ip_objects(body={}, method='DELETE', url_suffix=url_suffix, params={}, resp_type='content')
+            client.request_ip_objects(body={}, method='DELETE', url_suffix=url_suffix, params={
+                                      'list_target': list_target}, resp_type='content')
```

Comment:
It should always add the target list to the request? even in the allowlist? (you always pass the parameter but with changing values)
Can you add a comment explaining it?

Reply:
The `list_target` command argument only applies for denylist, it is ignored for allowlist (API behaviur).
The matching `f5-silverline-ip-object-add` command already implements this approach and adds deafult `list_target` to the request as well.
This is explained in the argument description, but I will add a comment to clarify.